tests/test_relevel.yml

```diff
@@ -3,12 +3,12 @@
   tags:
     - test_relevel
   files:
-    - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_genotype_WT_vs_KO.tsv
-      md5sum: 93fd663cb976c102144e1ffb34d88e53
+    - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_genotype_KO_vs_WT.tsv
```

Comment:
Why is it now `KO_vs_WT` while it ws the other way round before?

Reply:
So the relevel works now; before, it didn't. I have to say however that I noticed the order of factors is utterly confusing. I specifically looked this up; by default, orders of factors are alphabetically; after releveling, the reference should to my understanding be the first factor. This is not the case, but it does change compared to the normal test profile, I tested that

Reply:
Hmm I think that after releveling it should be `condition_of_interest_VS_condition_reference`, so the new naming should actually be correct.